loadflow/loadflow-validation/src/test/java/com/powsybl/loadflow/validation/BusesValidationTest.java

```diff
@@ -151,34 +151,34 @@ void setUp() throws IOException {
     @Test
     void checkBusesValues() {
         assertTrue(BusesValidation.INSTANCE.checkBuses("test", loadP, loadQ, genP, genQ, batP, batQ, shuntP, shuntQ, svcP, svcQ, vscCSP, vscCSQ,
-                                              lineP, lineQ, danglingLineP, danglingLineQ, t2wtP, t2wtQ, t3wtP, t3wtQ, mainComponent, looseConfig, NullWriter.NULL_WRITER));
+                                              lineP, lineQ, danglingLineP, danglingLineQ, t2wtP, t2wtQ, t3wtP, t3wtQ, mainComponent, looseConfig, NullWriter.INSTANCE));
```

Comment:
This refactoring has nothing to do in this PR (it slows down review) and should be reverted and done in another one.